### PR TITLE
feat: dawn-1 flame genesis file

### PR DESCRIPTION
**Pull request by quasystaty1**
adds dawn-1 flame `genesis.json`

### CODE REVIEW

File: dawn-1/evm/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Astria Dusknet 10 EVM
```
Comment (jbowen93) [inline dawn-1/evm/README.md]: Change to Dawn 1 EVM

File: dawn-1/evm/genesis.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      "shanghaiTime": 0,
+      "terminalTotalDifficulty": 0,
+      "terminalTotalDifficultyPassed": true,
+      "ethash": {},
+      "astriaOverrideGenesisExtraData": true,
+      "astriaSequencerInitialHeight": 65078,
+      "astriaRollupName": "flame-dawn-1",
+      "astriaCelestiaInitialHeight": 2951695,
+      "astriaCelestiaHeightVariance": 15000,
+      "astriaBridgeAddresses": [
```
Comment (jbowen93) [inline dawn-1/evm/genesis.json:23]: I'm assuming we'll update in a follow up PR to add the USDC bridge?

**Review by jbowen93 — APPROVED** [review]
LGTM but would appreciate clarity on USDC bridge address

File: dawn-1/evm/genesis.json (hunk 2 above)
Comment (quasystaty1) [reply dawn-1/evm/genesis.json:23]: correct.
Note that this genesis change does not affect block hashes in any way

**Pull request merged**